Web/Areas/Identity/Pages/_ValidationScriptsPartial.cshtml

```diff
@@ -2,7 +2,7 @@
         crossorigin="anonymous"
         integrity="sha256-+BEKmIvQ6IsL8sHcvidtDrNOdZO3C9LtFPtF2H0dOHI=">
 </script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validation-unobtrusive/3.2.11/jquery.validate.unobtrusive.min.js"
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validation-unobtrusive/3.2.12/jquery.validate.unobtrusive.min.js"
```

Comment:
Can we not upgrade to 4.0.0? Is there any breaking changes that affect us?

Reply:
I honestly have no idea, and i dont really have knowledge (or patience) to make sure it doesnt break anything. Further; two years has passed between version 3.2.12 and version 4.0.0 according to https://www.npmjs.com/package/jquery-validation-unobtrusive

Reply:
Fair enough - let's capture that this needs to be updated in a ticket; we might be able to get some input from Maciej or Sylwia.
I will approve - please make a ticket to update/investigate this.